build.gradle

```diff
@@ -64,11 +65,13 @@ configure(allProjs) {
         scalaGraphVersion = '1.12.5'
         scalafmtVersion = '1.5.1'
         hadoopVersion = 'hadoop2'
+        scalajCollVersion = '0.1.2'
         json4sVersion = '3.2.11' // matches Spark dependency version
         jodaTimeVersion = '2.9.4'
         jodaConvertVersion = '1.8.1'
         algebirdVersion = '0.13.4'
         jacksonVersion = '2.7.3'
+        jsoupVersion = '1.12.1'
```

Comment:
Can we worked it out with Jackson or Json4s without adding an additional dependency?

Reply:
We want a good java/scala libary to parse html, and based on my research Jsoup or HtmlCleaner came out as top options. https://tomassetti.me/parsing-html/  https://www.reddit.com/r/java/comments/4u8daa/best_html_parser_for_java/  Jsoup has better documentation and seems to be more widely-used than HtmlCleaner, hence the choice of Jsoup here.

Reply:
Have you looked into using Tika HTMLParser? https://tika.apache.org/0.7/formats.html#HyperText_Markup_Language

Reply:
Not sure which one brings less dependencies though and which one is more stable, but it would be nice to have the one with less dependencies and better support ;)